features/makephp.feature

```diff
@@ -259,3 +259,61 @@ Feature: Generate PHP files from PO files
       """
       new message
       """
+
+  Scenario: Should create pretty-printed PHP files
+    Given an empty foo-plugin directory
+    And a foo-plugin/foo-plugin-de_DE.po file:
+      """
+      # Copyright (C) 2018 Foo Plugin
+      # This file is distributed under the same license as the Foo Plugin package.
+      msgid ""
+      msgstr ""
+      "Project-Id-Version: Foo Plugin\n"
+      "Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/foo-plugin\n"
+      "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+      "Language-Team: LANGUAGE <LL@li.org>\n"
+      "Language: de_DE\n"
+      "MIME-Version: 1.0\n"
+      "Content-Type: text/plain; charset=UTF-8\n"
+      "Content-Transfer-Encoding: 8bit\n"
+      "POT-Creation-Date: 2018-05-02T22:06:24+00:00\n"
+      "PO-Revision-Date: 2018-05-02T22:06:24+00:00\n"
+      "X-Domain: foo-plugin\n"
+      "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+      #: foo-plugin.js:15
+      msgid "Foo Plugin"
+      msgstr "Foo Plugin"
+
+      #: foo-plugin.js:16
+      msgid "Hello"
+      msgstr "Hallo"
+
+      #: foo-plugin.js:17
+      msgid "You have %d new message"
+      msgid_plural "You have %d new messages"
+      msgstr[0] "Du hast %d neue Nachricht"
+      msgstr[1] "Du hast %d neue Nachrichten"
+      """
+
+    When I run `wp i18n make-php foo-plugin`
+    Then STDOUT should contain:
+      """
+      Success: Created 1 file.
+      """
+    And the return code should be 0
+    And the foo-plugin/foo-plugin-de_DE.l10n.php file should contain:
+      """
+      <?php
```

Comment:
Should not this be the output when `--pretty-print` is passed in `wp i18n make-php foo-plugin`?

Reply:
Not sure what you mean — we always need an opening PHP tag.

Reply:
`wp i18n make-php foo-plugin --pretty-print` should give this output but in line 299, `wp i18n make-php foo-plugin` is kept.

Reply:
Ah, I see. Indeed, the test should actually be using `--pretty-print`